lib/rex/post/meterpreter/packet.rb

```diff
@@ -314,6 +314,23 @@ def initialize(type, value = nil, compress=false)
     end
   end
 
+  def _tlv_type_string(value)
+    tlv_type = ::Rex::Post::Meterpreter::CommandMapper.find_cached_tlv_type_by_value(value)
+    ::Rex::Post::Meterpreter::CommandMapper.create_tlv_types_cache if tlv_type.nil?
+    tlv_type ||= ::Rex::Post::Meterpreter::CommandMapper.find_cached_tlv_type_by_value(value)
+
+    return if tlv_type.nil?
+
+    return_str = tlv_type.to_s.gsub('TLV_TYPE_', '').gsub('PACKET_TYPE_', '')
+
+    return return_str if tlv_type.is_a?(Symbol)
+    # In the off-chance we have multiple TLV types which have the same value,
+    # output them in the format `type=oneOf(A B C D)`
+    return "oneOf(#{return_str.gsub(/[:\[\] ]/, '').gsub(',', ' ')})" if tlv_type.is_a?(Array)
+
+    nil
```

Comment:
A method returning either a string or an array is a bit of a code smell; We could align on the command mapper returning an array of string for now to help simplify this logic

```suggestion
    tlv_names = ::Rex::Post::Meterpreter::CommandMapper.get_tlv_names(value).map { |name| name.gsub('TLV_TYPE_', '').gsub('PACKET_TYPE_', '') }
    
    case tlv_names.length
    when 0
        "unknown-#{value}"
    when 1
        tlv_names.first
    otherwise: 
       # In the off-chance we have multiple TLV types which have the same value
       "oneOf(#{tlv_names.join(",")})"
    end
```